h2d/Layers.hx

```diff
@@ -12,14 +17,25 @@ class Layers extends Object {
 		layerCount = 0;
 	}
 
+	/**
+		Adds a child object at the bottom layer (layer 0).
+	**/
 	override function addChild(s) {
 		addChildAt(s, 0);
 	}
 
+	/**
+	 * Shorthand for `addChildAt`, see `h2d.Layers.addChildAt`.
+	 */
 	public inline function add(s, layer) {
 		return addChildAt(s, layer);
 	}
 
+	/**
+	 * Adds a child `h2d.Object` at `layer:Int`.
+	 * @param s `h2d.Object` child to be added.
+	 * @param layer `Int` index of the layer, 0 is the bottom layer.
+	 */

```

Comment:
No need for documentation here. Should be mentioned in `add`

Reply:
Fixed.